ArduCopter/events.cpp

```diff
@@ -16,8 +16,8 @@ void Copter::failsafe_radio_on_event()
     } else {
         if (control_mode == AUTO && g.failsafe_throttle == FS_THR_ENABLED_CONTINUE_MISSION) {
             // continue mission
-        } else if (control_mode == LAND && g.failsafe_battery_enabled == FS_BATT_LAND && failsafe.battery) {
```

Comment:
You can't remove those battery conditions.

Reply:
I read this over like 20 times and didn't notice what was going on here. Fixed.